napari/layers/labels/labels.py

```diff
@@ -1001,6 +1007,9 @@ def _calculate_contour(
         )
         return sliced_labels[delta_slice]
 
+    def _get_cache_dtype(self, raw_dtype: np.dtype) -> np.dtype:
+        return _texture_dtype(len(self.colormap), raw_dtype)
+
```

Comment:
is this temporary to fix a bug and then should go away? If so, maybe put a comment so it's easy to find when we complete the deprecation.